business-list.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Business Table</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootswatch/3.3.7/yeti/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="business.css">
+</head>
+<body>
+<div class="container">
+  <div class="row">
+  <h1>Business Data</h1>
+     <table class="table table-striped table-hover">
+         <thead>
+               <tr>
+                   <th>Name</th>
+                   <th>Address</th>
+                   <th>Business</th>
+                </tr>
+          </thead>
+          <tbody>
+               <tr class="success">
+                  <td>Column content</td>
+                  <td>Column content</td>
+                  <td>Column content</td>
+                </tr>
+          </tbody>
+      </table> 
+
+  </div>
+</div>
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
```

Comment:
download the files from these links and add it in the project and give the local path to it